### DISCUSSION TITLE
Modification to Plotter Settings?

### DISCUSSION THREAD

**Discussion by kevbo049** [Chia-Network/chia-blockchain#6062] [Q&A]
Can we modify the memory allocation setting  while the plotter is plotting? I'm using the command line interface and currently have a k34 slowly chugging away.  While reading the wiki and other sources on plotting I realize the memory allocated was -b 8000.  I read the blog post from April regarding the new version which has DRAM allocation requirement for k34 at 14800.  My plot is currently on table 2 bucket 53.  Would it be possible to make an "in flight" change to the memory allocation or should I just scrap this plot altogether?  Thanks, in advance for your input and guidance.

**Comment by tiny0little** [comment]
unfortunately changing buffer size on fly not possible ... not sure they will ever implement something like this
